src/context/WalletProvider/MetaMask/config.ts

```diff
@@ -1,11 +1,19 @@
+import type { MetaMaskAdapter } from '@shapeshiftoss/hdwallet-metamask'
+import type { MetaMaskAdapter as MetaMaskMultiChainAdapter } from '@shapeshiftoss/hdwallet-shapeshift-multichain'
 import { getConfig } from 'config'
 import { MetaMaskIcon } from 'components/Icons/MetaMaskIcon'
 import type { SupportedWalletInfo } from 'context/WalletProvider/config'
 
-export const MetaMaskConfig: Omit<SupportedWalletInfo, 'routes'> = {
+type MetaMaskConfigType = Omit<
+  SupportedWalletInfo<typeof MetaMaskAdapter | MetaMaskMultiChainAdapter>,
+  'routes'
+>
+
+export const MetaMaskConfig: MetaMaskConfigType = {
   adapters: [
     {
       loadAdapter: () =>
+        // @ts-ignore TODO(gomes): FIXME
```

Comment:
Not happy with this, but probably our best bet a0622d87ef5e64d11877f039430c98a4398bb5a5